chirp-impl/src/main/java/sample/chirper/chirp/impl/JpaReadSide.java

```diff
@@ -0,0 +1,17 @@
+package sample.chirper.chirp.impl;
+
+import com.lightbend.lagom.javadsl.persistence.AggregateEvent;
+import com.lightbend.lagom.javadsl.persistence.ReadSideProcessor.ReadSideHandler;
+
+import javax.persistence.EntityManager;
+import java.util.function.BiConsumer;
+
+public interface JpaReadSide {
+    <Event extends AggregateEvent<Event>> ReadSideHandlerBuilder<Event> builder(String readSideId);
+
+    interface ReadSideHandlerBuilder<Event extends AggregateEvent<Event>> {
+        <E extends Event> ReadSideHandlerBuilder<Event> setEventHandler(Class<E> eventClass, BiConsumer<EntityManager, E> handler);
```

Comment:
Yeah, definitely. I only left them out here because I didn't need them yet, but I will include in the version pulled into the framework.